src/Modules/Cart/CartEndpoint.php

```diff
@@ -0,0 +1,104 @@
+<?php namespace Lean\Woocommerce\Modules\Cart;
+
+use Lean\AbstractEndpoint;
+
+/**
+ * Class CartEndpoint.
+ *
+ * @package Leean\Woocomerce\Modules\Cart
+ */
+class CartEndpoint extends AbstractEndpoint
+{
+	/**
+	 * Endpoint path
+	 *
+	 * @Override
+	 * @var String
+	 */
+	protected $endpoint = '/ecommerce/cart';
+
+	const POST_METHOD = 'POST';
```

Comment:
Ups, you're right. I read every of them except the `CREATABLE` I had `EDITABLE` first haha, thank you!